test/libsolidity/syntaxTests/modifiers/function_modifier_invocation_local_variables.sol

```diff
@@ -2,3 +2,5 @@ contract B {
     function f() mod(x) pure public { uint x = 7; }
     modifier mod(uint a) { if (a > 0) _; }
 }
+// ----
+// DeclarationError: (34-35): Undeclared identifier.
```

Comment:
Hm. This perhaps should be noted in the "breaking changes" section. We do not have one in the documentation yet..

Reply:
Should we have it in both `Language features` and `breaking changes` sections in the Changelog?

Reply:
Yes, both. Plus we should really have a section in the documentation I think - we can do that in a separate PR though.

Reply:
Done